product-service/src/main.ts

```diff
@@ -0,0 +1,60 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { Callback, Context, Handler } from 'aws-lambda';
+import serverlessExpress from '@vendia/serverless-express';
+
+let server: Handler;
+
+async function bootstrap(): Promise<Handler> {
+  const app = await NestFactory.create(AppModule, { cors: true });
+
+  /**Swagger*/
+  const config = new DocumentBuilder()
```

Comment:
I don't know `nestjs` but if it used only for creating swagger docs I'd recommend to look at `serverless-auto-swagger`.